auth_server/requirements.txt

```diff
@@ -0,0 +1 @@
+gitpython
```

Comment:
requirements.txt can now be removed

please rename README to README.md, otherwise GH doesn't seem to recognize markdown syntax